beamer/cli.py

```diff
@@ -51,6 +51,11 @@ def _sigint_handler(agent: Agent) -> None:
     type=int,
     help="Time in seconds to wait for a fill event before challenging a false claim.",
 )
+@click.option(
+    "--unsafe-fill-time",
+    type=int,
+    help="Time in seconds to ignore requests according to closeness of request expiry",
```

Comment:
Maybe we should reuse the line from docs here for the help string, it's not much longer:
```Time in seconds before request expiry, during which the agent will consider it unsafe to fill and ignore the request.
```